internal/exec/describe_component.go

```diff
@@ -149,6 +149,8 @@ func ExecuteDescribeComponent(
 	var configAndStacksInfo schema.ConfigAndStacksInfo
 	configAndStacksInfo.ComponentFromArg = component
 	configAndStacksInfo.Stack = stack
+	configAndStacksInfo.CliCommand = "describe"
+	configAndStacksInfo.CliSubCommand = "component"
```

Comment:
IMO it would be better to just expose the positional arguments and flags, rather than labeling them as command subcommand and subcommand2. It has a code smell to me, like naming something "new2".

I am okay with cli_command and cli_subcommand, but everything after that shouldn't be numbered. It should be an array.